src/access/ProxyOwner.sol

```diff
@@ -1,50 +1,21 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.13;
 
-import "@openzeppelin/contracts/access/Ownable2Step.sol";
-import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import { Ownable, Ownable2Step } from "@openzeppelin/contracts/access/Ownable2Step.sol";
+import { ProxyAdmin } from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
 
 /**
  * @title ProxyOwner
  * @notice ProxyAdmin with 2-step ownership transfer
- * @dev Adds 2-step ownership transfer to the OpenZeppelin ProxyAdmin contract, both for the owner of the ProxyAdmin
- *      and to the proxy ownership transfer.
+ * @dev Adds 2-step ownership transfer to the OpenZeppelin ProxyAdmin contract for the owner of the ProxyAdmin.
```

Comment:
This is actually a lot cleaner / glad they made admin's immutable in the proxy layer.